processor/spanmetricsprocessor/processor.go

```diff
@@ -51,6 +51,9 @@ const (
 
 	defaultDimensionsCacheSize         = 1000
 	defaultResourceAttributesCacheSize = 1000
+
+	defaultCallsTotalMetricName = "calls_total"
+	defaultLatencyMetricName    = "latency"
```

Comment:
if we create this processor starts from scratch, will we still call it `latency`? As the metrics are not only HTTP metrics, `duration` probably makes more sense.
As it's already named as this from the upstream, we can just leave it as it is.

Reply:
I think `latency` can still be ok. You can still have database latency (time it takes to get back to you), network latency, etc.. I honestly don't feel strongly about it though, I think `latency` and `duration` can both work.

Reply:
The ideal end state is a mapping from the trace convention to the metrics convention. e.g. `http.client.duration -> metric.http.client.duration`. Not asking for that here, but that's what we should be looking to eventually do.

Reply:
I think this default name should mostly for custom traces that don't follow a convention. Although ideally we really shouldn't be doing anything custom but I think that'll be a hard ask 😅 somewhere along the way someone will probably want something custom and have some valid reason for it